ibis/backends/pyspark/__init__.py

```diff
@@ -1286,7 +1286,7 @@ def _to_filesystem_output(
             df = df.write.format(format)
             for k, v in (options or {}).items():
                 df = df.option(k, v)
-            df.save(path)
+            df.save(os.fspath(path))
```

Comment:
Is this code actually being tested anywhere specific, or does it get hit as part of another code path?

Reply:
This wasn't tested before, but I believe it gets hit in the test I'm running, because before this the test failed in the read parquet part. Maybe @chloeh13q has more context

Reply:
Ah, it's that pyspark doesn't support `Path` objects 🙄